lib/suspenders/app_builder.rb

```diff
@@ -240,6 +240,17 @@ def create_database
       bundle_command 'exec rake db:create db:migrate'
     end
 
+    def replace_gemfile(path)
+      remove_file 'Gemfile'
```

Comment:
Dunno; you tell me.

I always follow the existing coding style. A mix of coding style throughout a codebase seems odd to me. Maybe we should make a big commit to change this code style?